src/Core/Request/Customers/CustomerLoginRequest.php

```diff
@@ -67,16 +69,19 @@ class CustomerLoginRequest extends AbstractApiRequest
     /**
      * @param string $email
      * @param string $password
-     * @param string $anonymousCartId
+     * @param string $anonymousCart
```

Comment:
please update the param here

```suggestion
     * @param CartReference|string $anonymousCart
```

Reply:
yes, I didn't see it ;)